bootcamp/tutorials/quickstart/hybrid_search_with_milvus.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<p>this code block doesn't generate embeddings, it just creates a function. the last line of code shall use random_embedding() to generate docs_embeddings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>